graylog2-server/src/main/java/org/graylog/security/authservice/backend/ADAuthServiceBackend.java

```diff
@@ -257,13 +258,17 @@ private ADAuthServiceBackendConfig buildTestConfig(@Nullable AuthServiceBackendD
                 .put("login_success", loginSuccess);
 
         if (user != null) {
-            userDetails.put("user_details", ImmutableMap.of(
-                    "dn", user.dn(),
-                    AD_OBJECT_GUID, user.base64UniqueId(),
-                    config.userNameAttribute(), user.username(),
-                    config.userFullNameAttribute(), user.fullName(),
-                    "email", user.email()
-            ));
+            userDetails.put("user_details", ImmutableMap.<String, String>builder()
+                    .put("dn", user.dn())
+                    .put("account_disabled", String.valueOf(user.userAccountControl().accountIsDisabled()))
+                    .put("normal_user_account", String.valueOf(user.userAccountControl().isUserAccount()))
+                    .put("password_expired", String.valueOf(user.userAccountControl().passwordExpired()))
```

Comment:
How about only putting in the `userAccountControl` attribute as key and then all values as named flags?

```
"userAccountControl": "ACCOUNTDISABLED|NORMAL_ACCOUNT|PASSWORD_EXPIRED"
```
I think that would make it a bit clearer to AD admins. `account_disabled`, `normal_user_account` and `password_expired` are not really AD attributes.

Reply:
sure, I can do that.